lambda-pages/src/views/NoteView.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { getNote, deleteNote } from '../actions';
+import styled from 'styled-components';
+import Note from '../components/Note';
+import NoteNav from '../components/NoteNav';
+import Modal from '../components/Modal';
+import Portal from '../components/Portal';
+import StyledSpinner from '../styles/StyledSpinner';
+
+const StyledNote = styled.div`
+  background: #fff;
+  padding: 2rem;
+  border-radius: 5px;
+  margin-bottom: 5rem;
+  box-shadow: 0 3px 6px -3px rgba(0, 0, 0, 0.7);
+
+  h2 {
+    font-size: 4.2rem !important;
+    margin-bottom: 1rem !important;
+  }
+
+  hr {
+    margin-bottom: 3rem !important;
+  }
+`;
+
+class NoteView extends Component {
+  state = {
+    id: this.props.match.params.id,
```

Comment:
does this really need to be on state? Normally something should only be on state if it's being updated by the component's own methods during its lifecycle